mods/carts/rails.lua

```diff
@@ -0,0 +1,68 @@
+minetest.register_node("carts:rail", {
```

Comment:
Could be registered with `carts:register_rail` aswell.
The default rail groups are always the same (see function `carts:get_rail_groups()`) so additional groups in the rail definition could be adapted to those in the register function.